realm/realm-library/src/objectServer/java/io/realm/mongodb/Credentials.java

```diff
@@ -99,23 +116,25 @@ public static Credentials apple(String idToken) {
     }
 
     /**
-     * FIXME
+     * Creates credentials representing a remote function previously added to MongoDB Realm. The
+     * input parameters
      * <p>
      * This provider must be enabled on MongoDB Realm to work.
      *
+     * @param arguments {@link Document} containing the values passed to the remote function. The
+     *                  keys must match the names given to the remote function parameters.
```

Comment:
This is a bit confusing to me. I would consider either removing it completely or rewording it. Maybe to something like `The document will be parsed as an argument to the remote function, so the keys must match the format the function expects.`